packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -1384,3 +1384,9 @@ export const formatDate = (date: Date, useUTC?: boolean) => {
   const timeFormat = useUTC ? d3UtcFormat : d3TimeFormat;
   return timeFormat('%-e %b %Y, %H:%M')(date) + (useUTC ? ' GMT' : '');
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function getSecureProps(props: Record<string, any> = {}): Record<string, any> {
+  const { dangerouslySetInnerHTML, ...result } = props;
```

Comment:
can other IChartProps like chartTitle cause XSS vulnerability? In that case shall we also sanitize any strings that we get as input?

Reply:
No. React takes care of them. See this: https://github.com/microsoft/fluentui/pull/33368#issuecomment-2507601147